packages/rx_bloc_list/test/stub.dart

```diff
@@ -49,7 +49,7 @@ class Stub {
   }
 }
 
-class IdentifiableModel implements Identifiable {
```

Comment:
What is the reason of using extends instead of implements ?

Reply:
Because `isEqualToIdentifiable` method inside `Identifiable` is implemented.
You can use either `implements` or `with`.